src/main/java/com/basho/riak/client/core/RiakNode.java

```diff
@@ -640,18 +650,23 @@ private Channel getConnection()
         {
             try
             {
-                channel = doGetConnection();
+                channel = doGetConnection(true);
```

Comment:
Argh stupid GH code folding. Even when I expanded the diff it still wasn't clear. Sorry!